.github/workflows/main.yml

```diff
@@ -48,9 +48,9 @@ jobs:
         run: |
           python -m pylint src/gstools/
 
-      - name: cython-lint check
-        run: |
-          cython-lint src/gstools/
+      #- name: cython-lint check
+        #run: |
+          #cython-lint src/gstools/
```

Comment:
This should be solved with #343

Reply:
I'll solve that in a different PR.

Reply:
That's going to be solved in PR #354.

Reply:
PR #354 should be merged first and then I can update this branch with the fixes.